kcidb/db/bigquery/v04_00.py

```diff
@@ -504,10 +504,13 @@ class Schema(AbstractSchema):
         checkout="SELECT\n"
                  "    id,\n"
                  "    git_commit_hash,\n"
+                 "    NULL AS git_commit_tags,\n"
+                 "    NULL AS git_commit_message,\n"
```

Comment:
I think it would be valuable to expose these two fields in "revisions" as well, but the aggregation expressions [might not be so trivial across all the databases](https://chatgpt.com/share/017c66e8-a332-4157-bc25-6b12f98903cd) and this would need some extra testing. So let's leave that for when we need it.